concepts/05 UI Components/DataGrid/10 Enhance Performance on Large Datasets/030 Rendering Optimization.md

```diff
@@ -1,7 +1,22 @@
-The UI component renders all rows and columns once the data is loaded. To improve performance, set the [rowRenderingMode](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/scrolling/rowRenderingMode.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/scrolling/#rowRenderingMode') and [columnRenderingMode](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/scrolling/columnRenderingMode.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/scrolling/#columnRenderingMode') properties in the **scrolling** object to *"virtual"*. These properties specify that UI elements are only rendered when they come into the viewport.
+If you work with a large data set, we recommend that you apply these settings to improve rendering performance:
```

Comment:
"... apply these settings ..." - list items here are not settings.
"... to improve rendering performance ..." - the word "improve" means that something was in bad condition and then it became better. Generally, prefer "enhance". In this example, "optimize" fits even better.

```suggestion
If you work with a large data set, apply the following recommendations to optimize rendering performance:
```